next/pages/mestske-sluzby/dev/[slug].ts

```diff
@@ -1,63 +1,39 @@
-import komunitneZahrady from '@forms-shared/definitions/komunitne-zahrady'
-import predzahradky from '@forms-shared/definitions/predzahradky'
-import priznanieKDaniZNehnutelnosti from '@forms-shared/definitions/priznanie-k-dani-z-nehnutelnosti'
-import stanoviskoKInvesticnemuZameru from '@forms-shared/definitions/stanovisko-k-investicnemu-zameru'
-import zavazneStanoviskoKInvesticnejCinnosti from '@forms-shared/definitions/zavazne-stanovisko-k-investicnej-cinnosti'
-import { RJSFSchema, UiSchema } from '@rjsf/utils'
+import { getFormDefinitionBySlug } from '@forms-shared/definitions/form-definitions-helpers'
 
 import FormPageWrapper, { FormPageWrapperProps } from '../../../components/forms/FormPageWrapper'
 import { SsrAuthProviderHOC } from '../../../components/logic/SsrAuthContext'
 import { environment } from '../../../environment'
 import { amplifyGetServerSideProps } from '../../../frontend/utils/amplifyServer'
 import { slovakServerSideTranslations } from '../../../frontend/utils/slovakServerSideTranslations'
 
-const slugSchemasMap = {
-  'priznanie-k-dani-z-nehnutelnosti': priznanieKDaniZNehnutelnosti,
-  'stanovisko-k-investicnemu-zameru': stanoviskoKInvesticnemuZameru,
-  'zavazne-stanovisko-k-investicnej-cinnosti': zavazneStanoviskoKInvesticnejCinnosti,
-  'komunitne-zahrady': komunitneZahrady,
-  predzahradky,
+type Params = {
+  slug: string
 }
 
 /**
  * A route to preview forms in `forms-shared` folder. Backend functionality doesn't work. Works only in development.
  */
-export const getServerSideProps = amplifyGetServerSideProps<FormPageWrapperProps>(
+export const getServerSideProps = amplifyGetServerSideProps<FormPageWrapperProps, Params>(
```

Comment:
is this page still relevant with the new approach ?

Reply:
Yes, it's useful.